docs/jwt-auth.md

```diff
@@ -0,0 +1,76 @@
+# Bot Authentication for third-party API's
+
+There may be a case where you want your detection bot to make an external call to an API you own. The Forta bot sdk provides the following methods for generating a JWT token which allows bots make authorized requests to external APIs by using the scan node's identity which the detection bot is running on:
+
+- `fetchJwtToken(claims, expiresAt)` See method details [javascript/typescript](sdk.md#fetchjwttoken),[python](python.md#fetchjwttoken)
+- `decodeJwtToken(token)` See method details [javascript/typescript](sdk.md#decodejwttoken),[python](python.md#decodejwttoken)
+
+!!! warning "It is up to the external API to verify the JWT & verify the smart contract in the JWT data"
+
+## Detection Bot Authentication
+
+Suppose you own an api service and you want your detection bot to have access to your api service. You can generate a JWT token which will contain the following additional fields in it's data:
+
```

Comment:
Could we add an example of how the generated JWT token can be used to access an example API service? I'm not sure what happens after the token is generated. Maybe one line of code that makes a request to a fake API with the token in the payload?

Reply:
Added and express server example.